convert2rhel.ini.example

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Example of configuration file convert2rhel.ini for secrets.
+# Possible locations of this file:
+# 1) user specified and passed by -c, --config-file option; highest priority
+# 2) ~/convert2rhel.ini; lower priority
+# 3) /etc/convert2rhel.ini; the lowest priority
+
+# [SECRET]
+# password = <insert_password>
+# activation-key = <insert_activation_key>
```

Comment:
I agree. We can put it to the existing `convert2rhel/data/version-independent/`. I don't think another directory is needed.

Reply:
I wonder in general if we should move `convert2rhel/data` somewhere else.  setup.py installs that directory into `site-packages/convert2rhel` and I see that we then delete it from there inside of the rpm spec file.

Reply:
This file is also different from the other files in convert2rhel/data since it:
* Is meant to be user edited to become useful in the common case
* Should be installed with mode 0600 for security
* Installs to /etc/ rather than /usr/share/

I wonder if all of those reasons make it better to place it in a different directory than convert2rhel/data ?

Reply:
From the meeting today,
- We'll keep the files in convert2rhel/data/ and the config file separate.
- Moving convert2rhel/data will be a separate issue
- Place the data file in a subdirectory of the toplevel

Changes to make:
- Store the sample config file in `config/convert2rhel.ini`
- Modify the `MANIFEST.in` to put the config file in the tarball:
  ``` patch
  + include config/convert2rhel.ini
  ```
- Modify the rpm spec file (`packaging/convert2rhel.spec`) to include the config file.  A spec file has  three main parts: Some metadata, shell scripts to control building and installing, and a file          manifest.  We'll need to modify the shell script and the file manifest.
        
  At the end of the `%install` section, add a comment explaining what we're doing and the shell command to install the file.  We prepend the path to install to with `%{buildroot}` because rpm looks for the files to install in a parallel directory tree, the buildroot, rather than directly using what is in `/`.  `%{_sysconfdir}` is an rpm macro that expands to `/etc`. 
  ```                                            
  # Put the config file into place
  install -m 0600 config/convret2rhel.ini %{buildroot}%{_sysconfdir}/convert2rhel.ini          
  ```                                                                                                 
           
  In the `%files` section, add the config file:
  ``` patch
    %{_sharedstatedir}/%{name}
  + %config(noreplace) %attr(0600, root, root) %{_sysconfdir}/convert2rhel.ini
  ```